src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/Buttons/BitCompoundButtonDemo.razor.cs

```diff
@@ -180,39 +180,101 @@ public partial class BitCompoundButtonDemo
     };
 
     private readonly string example1HTMLCode = @"
-<BitCompoundButton IsEnabled=""true""
-                   Text=""Primary""
-                   SecondaryText=""This Button is a compound button"">
-</BitCompoundButton>
-<BitCompoundButton IsEnabled=""true""
-                   Text=""Standard""
-                   AriaHidden=""true""
-                   SecondaryText=""This Button is a compound button""
-                   ButtonStyle=""BitButtonStyle.Standard"">
-</BitCompoundButton>
-<BitCompoundButton IsEnabled=""false""
-                   AllowDisabledFocus=""false""
-                   Text=""Disabled""
-                   Class=""disable-cmp-btn""
-                   AriaDescription=""Detailed description used for screen reader.""
-                   SecondaryText=""This Button is a disabled compound button"">
-</BitCompoundButton>";
+<style>
+    .buttons-container {
+        display: flex;
+        flex-flow: row wrap;
+        gap: 0.5rem;
+    }
+</style>
+
+<div class=""buttons-container"">
+    <BitCompoundButton IsEnabled=""true""
+                        Text=""Primary""
+                        SecondaryText=""This Button is a compound button"">
+    </BitCompoundButton>
+    <BitCompoundButton IsEnabled=""true""
+                        Text=""Standard""
+                        AriaHidden=""true""
+                        SecondaryText=""This Button is a compound button""
+                        ButtonStyle=""BitButtonStyle.Standard"">
+    </BitCompoundButton>
+    <BitCompoundButton IsEnabled=""false""
+                        AllowDisabledFocus=""false""
+                        Text=""Disabled""
+                        Class=""disable-cmp-btn""
+                        AriaDescription=""Detailed description used for screen reader.""
+                        SecondaryText=""This Button is a disabled compound button"">
+    </BitCompoundButton>
+</div>";
 
     private readonly string example2HTMLCode = @"
-<BitCompoundButton Style=""height: 80px;font-size: 16px;text-decoration: underline;"" Text=""Styled"" SecondaryText=""This is styled compound button""></BitCompoundButton>
-<BitCompoundButton ButtonStyle=""BitButtonStyle.Standard"" Class=""custom-compound-button"" Text=""Classed"" SecondaryText=""This is classed compound button""></BitCompoundButton>";
+<style>
+    .buttons-container {
+        display: flex;
+        flex-flow: row wrap;
+        gap: 0.5rem;
+    }
+
+    .custom-compound-button.standard {
+        width: rem(230px);
```

Comment:
plz don't use these scss mixins (like rem here) in the examples and their sample codes